src/layouts/BaseLayout/hooks/NavigatorDataQueryOpts.ts

```diff
@@ -0,0 +1,91 @@
+import { queryOptions as queryOptionsV5 } from '@tanstack/react-queryV5'
+import { z } from 'zod'
+
+import {
+  RepoNotFoundErrorSchema,
+  RepoOwnerNotActivatedErrorSchema,
+} from 'services/repo'
+import Api from 'shared/api'
+import { rejectNetworkError } from 'shared/api/helpers'
+
+const RepositorySchema = z.object({
+  __typename: z.literal('Repository'),
+  name: z.string(),
+})
+
+const RequestSchema = z.object({
+  owner: z
+    .object({
+      isCurrentUserPartOfOrg: z.boolean(),
+      repository: z.discriminatedUnion('__typename', [
+        RepositorySchema,
+        RepoNotFoundErrorSchema,
+        RepoOwnerNotActivatedErrorSchema,
+      ]),
+    })
+    .nullable(),
+})
+
+const query = `
+query NavigatorData($username: String!, $repo: String!) {
+  owner(username: $username) {
+    isCurrentUserPartOfOrg
+    repository(name: $repo) {
+      __typename
+      ... on Repository {
+        name
+      }
+      ... on NotFoundError {
+        message
+      }
+      ... on OwnerNotActivatedError {
+        message
+      }
+    }
+  }
+}`
+
+interface NavigatorDataQueryArgs {
+  provider: string
+  owner: string
+  repo: string
+}
+
+export const NavigatorDataQueryOpts = ({
```

Comment:
Why are these lines not covered? Tests failing or something?

Reply:
It looks like the CI hasn't ran or something